examples/karpenter/README.md

```diff
@@ -72,19 +73,11 @@ You should see one Self-managed node up and running
       karpenter-controller-5f959cdc44-8dmjb   1/1     Running   0          31m
       karpenter-webhook-65f48f8d49-5hkpb      1/1     Running   0          31m
 
-#### Step8: Deploy the default provisioner
-Kaprpenter will be ready to spin up SPOT/ON-DEMAND nodes based on the provided configuraiton in `default_provisioner.yaml`
-
-    $ cd examples/eks-cluster-with-karpenter/provisioners
-    $ kubectl apply -f default_provisioner.yaml
-
-#### Step9: Run this sample `deployment` to verify the Autoscaling triggered by Karpenter
-
-    $ cd examples/eks-cluster-with-karpenter/provisioners
-    $ kubectl apply -f sample_deployment.yaml
+# How to Destroy
 
+NOTE: Make sure you delete all the deployments which clean up the nodes spun up by Karpenter Autoscaler
```

Comment:
This is an important thing to discuss. Terraform does not address the drift when using Karpenter.

Reply:
That's why added this note to the users.
We should be ok as Terraform only cares about the nodes created by Terraform Node groups. It will not have any drift issues with Karpenter nodes as these are not joining the existing node groups.

Karpenter Nodes can only be deleted when you delete the deployments.